ops/jujuversion.py

```diff
@@ -55,28 +56,29 @@ def __repr__(self):
             s += '.{}'.format(self.build)
         return s
 
-    def __eq__(self, other):
+    def __eq__(self, other: object) -> bool:
```

Comment:
Would the following suggested change not obviate the need for the `cast`? (applies to the `__lt__` dunder too)

```suggestion
    def __eq__(self, other: Union[str, "JujuVersion"]) -> bool:
```